### DISCUSSION TITLE
Instant Login

### DISCUSSION THREAD

**Discussion by flexminister** [kimai/kimai#2803] [Questions & Answers] [answered]
Hello everybody, 

im absolutely new in php development. But i was able to make a bundle which puts additional headers to my session. Now i have to trigger the login process with these header values. But i have no clue how to do it. Can anybody help me out?

**Comment by kevinpapst** [comment, marked as answer]
This is likely what you want: https://github.com/kevinpapst/kimai2/blob/master/src/Security/TokenAuthenticator.php

But bear in mind, this has almost nothing to do with Kimai. Kimai uses Symfony, so you want to read about their security and authentication system.

**Reply by flexminister** [reply]
Thanks for the tip. I will learn more about symfony then.
